pytest_asyncio/plugin.py

```diff
@@ -49,6 +49,16 @@
     PytestPluginManager,
 )
 
+_seen_markers: set[int] = set()
```

Comment:
I assume this code is intended to suppress duplicate warnings about keyword arguments in the *asyncio* marker. This should no longer be needed, because it has been addressed in https://github.com/pytest-dev/pytest-asyncio/pull/1203/commits/8585040230a04c512053db9c64d4410121a4ff58.